src/main/java/org/apache/maven/plugins/changelog/ChangeLogReport.java

```diff
@@ -283,6 +283,12 @@ public class ChangeLogReport extends AbstractMavenReport {
     @Parameter(defaultValue = "connection", required = true)
     private String connectionType;
 
+    /**
+     * If true, file and file revision information is hidden for each SCM entry.
+     */
+    @Parameter(property = "hideFileAndRevInfo", defaultValue = "false")
```

Comment:
avoid abbreviations. That is, hideFileAndRevision

Is it hidden or omitted?

Reply:
Thanks for your feedback, I've changed it to `omitFileAndRevision`. I think it's much clearer now